utils/setools/patches/020-no-deprecated.patch

```diff
@@ -0,0 +1,10 @@
+--- a/setup.py
++++ b/setup.py
+@@ -126,6 +126,7 @@ ext_py_mods = [Extension('setools.policyrep', ['setools/policyrep.pyx'],
+                                              '-Wno-unreachable-code',
+                                              '-Wno-implicit-fallthrough',
+                                              '-Wno-cast-function-type',
++                                             '-Wno-deprecated-declarations',
```

Comment:
Why this was not submitted to upstream? There is no commit message in this patch.

Reply:
This is not a proper fix. The issue is that setools uses APIs deprecated by Python3. The only reason the build fails is because it passes -Werror.

Reply:
Which one? This is a missing here. You are doing some changes, which are not documented. Neither reported to upstream to issue.

Reply:
No idea which APIs are used. I don't care much for python.

Upstream will find out eventually. Python 3.9 only recently got released.

Reply:
This is not a good approach.

Reply:
See also #13762 - I'm not a fan of just getting things to compile without making sure it works / fixing the underlying issue.

Reply:
I would be rather if I don't know about that. As this is such bad.